TwoPageEBookReader/app/src/main/java/com/example/ebook_reader_sample/StartActivity.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ */
+package com.example.ebook_reader_sample
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+
+class StartActivity : AppCompatActivity(){
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_start)
+
+
+        setSupportActionBar(findViewById(R.id.toolbar))
+
+        val titleRecyclerView = findViewById<RecyclerView>(R.id.title_recycler)
+        titleRecyclerView.layoutManager = LinearLayoutManager(this)
```

Comment:
minor detail, but you might consider making the "books" subdirectory name a `const`

Reply:
done